source/includes/steps-export.yaml

```diff
@@ -7,36 +8,57 @@ content: |
   left-hand pane.
 
 ---
-title: Click :guilabel:`Collection` in the top-level menu and select :guilabel:`Export Collection`
+title: Click :guilabel:`Collection` in the top-level menu and select 
+       :guilabel:`Export Collection`
 level: 4
 ref: export-collection
 content: |
-
-  |
-
   .. figure:: /images/compass/export-data-option-select.png
      :alt: Export data select menu
 
   |compass-short| displays the following dialog:
 
-  .. figure:: /images/compass/export-data-dialog.png
-     :alt: Export data dialog
-     :figwidth: 600px
+  .. figure:: /images/compass/export-data-dialog-1.png
+     :alt: Export data dialog, step 1
+     :width: 60%
+
+  The export dialog initially displays the query entered in the query 
+  bar prior to export, if applicable. If no query was specified, this 
+  section displays a generic ``find`` operation.
```

Comment:
I don't love "generic" find operation. Let's be specific about what happens. "This section displays a ``find`` operation with no parameters, which returns all documents in the collection."